packages/core/src/tools/tool-builder/builder.ts

```diff
@@ -52,6 +52,15 @@ export class CoreToolBuilder extends MastraBase {
     return this.originalTool.inputSchema ?? z.object({});
   };
 
+  private getOutputSchema = () => {
+    // Vercel tools don't support output schemas yet, this is a placeholder for when they do.
+    if (isVercelTool(this.originalTool)) {
+      return (this.originalTool as any).outputSchema;
```

Comment:
If they don't support it wont it be undefined here? Probably clearer to do:
```suggestion
      return null;
```

Reply:
_or_ how about `if ('outputSchema' in this.originalTool) return this.originalTool.outputSchema`